homeassistant/components/waqi/sensor.py

```diff
@@ -73,7 +73,7 @@
             stations = await client.search(location_name)
             _LOGGER.debug("The following stations were returned: %s", stations)
             for station in stations:
-                waqi_sensor = WaqiSensor(client, station)
+                waqi_sensor = WaqiSensor(client, station, token)
                 if not station_filter or {
                     waqi_sensor.uid,
```

Comment:
This seems like it is a unique ID ?

Reply:
This is actually a 3 digit code (at least for my own station).  I will combine this with the token to make it unique.